inspirehep/modules/records/api.py

```diff
@@ -24,45 +24,92 @@
 
 from __future__ import absolute_import, division, print_function
 
+import copy
 from datetime import datetime
 
 import arrow
 from elasticsearch.exceptions import NotFoundError
 from flask import current_app
 
 from inspire_schemas.api import validate
+from inspire_schemas.builders import LiteratureBuilder
 from invenio_files_rest.models import Bucket
 from invenio_pidstore.errors import PIDDoesNotExistError
 from invenio_pidstore.models import PersistentIdentifier
 from invenio_records_files.api import Record
 from invenio_db import db
 
+from inspirehep.modules.records.utils import open_url_or_path
 from inspirehep.utils.record_getter import (
     RecordGetterError,
     get_es_record_by_uuid
 )
 
 
+MAX_UNIQUE_KEY_COUNT = 50000
+
+
 class InspireRecord(Record):
 
     """Record class that fetches records from DataBase."""
 
+    @classmethod
+    def create(cls, *args, **kwargs):
+        """
```

Comment:
Something's missing here!

Reply:
what do you mean?

Reply:
From https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings:
> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

The summary line is missing.

Reply:
I will not add a spurious one line comment there because some 'directive' says it should. Imo we are better without comments than with "Creates a record." comment. That's why we are not using the docstring checker invenio uses.

Reply:
From that same page:
```
The aim of this PEP is to standardize the high-level structure of docstrings: what they should contain, and how to say it (without touching on any markup syntax within docstrings). The PEP contains conventions, not laws or syntax.
```

Reply:
> Imo we are better without comments than with "Creates a record." comment.

And I agree with that, because this is not what it should say! Wouldn't it be better to have
```
"""Override the ``create`` method.

[explanation of what is wrong with the overridden method]

Args:
    [...]
"""
```
?

Reply:
The first phrase there still does not say anything...